remotecommanddispatcher/src/main/java/com/tealium/remotecommanddispatcher/remotecommands/JsonRemoteCommand.kt

```diff
@@ -1,11 +1,11 @@
 package com.tealium.remotecommanddispatcher.remotecommands
 
+import com.tealium.core.TealiumConfig
 import com.tealium.remotecommanddispatcher.RemoteCommandConfigRetriever
-import com.tealium.remotecommands.RemoteCommand
 
-abstract class JsonRemoteCommand(val id: String,
-                                 val detail: String?,
+internal class JsonRemoteCommand(val config: TealiumConfig,
```

Comment:
Android Studio has `filename` and `remoteUrl` as unreferenced from this class so guess we can remove them.
I can't find any usages for the `config` or `id` param either - are we able to get rid of this class altogether and just store the config retriever?